e2e/suite_test.go

```diff
@@ -96,8 +96,8 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	gomega.Expect(secretCreateErr).ShouldNot(gomega.HaveOccurred())
 
 	attachedCreateErr := resources.CreateAttachedCluster(kuratorClient, attachedcluster)
-	gomega.Expect(attachedCreateErr).ShouldNot(gomega.HaveOccurred())
 	resources.WaitAttachedClusterFitWith(kuratorClient, namespace, memberClusterName, func(attachedCluster *clusterv1a1.AttachedCluster) bool {
```

Comment:
Nit: maybe we can rename WaitAttachedClusterFitWith to WaitAttachedClusterReady

if we want to check other status, we can provide a function like `waitUntil(context, func()error)`